### PR TITLE
24 feat add n00bz writeup

### PR DESCRIPTION
**Pull request by Mathis-Dory**
### Added

* N00BZCTF 2023 index
* H0N3YP0T writeups ( all OSINT chall)
* Sunbather writeups

### CODE REVIEW

File: content/n00bzctf_2023/johnDoesStrikesAgain.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+key = "YouCanNeverCatchJohnDoe!"
+message = b'\x13\x00\x1d-A*!\x00Q\x16R\x02\x12\x07\n\x1b>\x0e\x06\x1a~O-D CU\t\x0e\x06 E2\n\x17bA#\x0b\t>O\x11\x011O\tH*\x1b\x10-\x08\x00)E\x02\nMck~)\x07"\x01H*+\n_\x01\x00\x00\x00c\n\x00!\x12V\r\x1d4A\x19\x16\x0b"O!N(\x00\x13Dy\x02\x000\x08\rn\x16\x19E\x16,\x0fS\x17H+\x1c\x03N)\nEU1\x0e\x01c\x10\x1b+\x16\x02\x0c\x1d-A\x11\x15\r8\x16H\x0f#\x0e\x0cOx'
+
+decrypted_message = bytearray([m ^ k for (m, k) in zip(message, itertools.cycle(key.encode()))])
+
+print(decrypted_message.decode())
+
+```
+
+The output is the following: `John Doe: You know how much I love music so don't ask me that question every again! `
```
Comment (Stefan-Radu) [inline content/n00bzctf_2023/johnDoesStrikesAgain.md:40]: the only thing, is that this inline code has a trailing space at the end lol

**Comment by Stefan-Radu** [comment]
@Mathis-Dory looks good to me. I had just a really small nitpick comment.

**Review by Stefan-Radu — CHANGES_REQUESTED** [review]

File: content/n00bzctf_2023/johnDoesStrikesAgain.md (hunk 1 above)
Comment (Mathis-Dory) [reply content/n00bzctf_2023/johnDoesStrikesAgain.md]: Thx fixed :)

Comment (Mathis-Dory) [reply content/n00bzctf_2023/johnDoesStrikesAgain.md]: @Stefan-Radu you have the approved the changement I made so I can merge

**Review by Stefan-Radu — APPROVED** [review]

**Comment by Mathis-Dory** [comment]
PR closed but if you guys want to add you writeup you can reopen it :)

**Pull request merged**